runtime/drive_handler_test.go

```diff
@@ -95,7 +95,7 @@ func TestContainerStubs(t *testing.T) {
 		mockMachine, err := firecracker.NewMachine(ctx, firecracker.Config{}, firecracker.WithClient(
 			firecracker.NewClient("/path/to/socket", nil, false, firecracker.WithOpsClient(&fctesting.MockClient{
 				PatchGuestDriveByIDFn: func(params *ops.PatchGuestDriveByIDParams) (*ops.PatchGuestDriveByIDNoContent, error) {
-					assert.Equal(t, hostPath, firecracker.StringValue(params.Body.PathOnHost))
+					assert.Equal(t, hostPath, firecracker.StringValue(&params.Body.PathOnHost))
```

Comment:
How about create a new PR for this change. What do you think @kzys ?

Reply:
This error shows up after updating to most recent go-sdk.  Yes we can make a separate PR but not sure how different it will be. Right now keeping it as a separate commit.

Reply:
Yeah, I just saw your separate commit for this. It is enough though imo.